lib/chrono_model/adapter/ddl.rb

```diff
@@ -78,8 +78,9 @@ def remove_history_validity_constraint(table, options = {})
         # allow setting the PK to a specific value (think migration scenario).
         #
         def chrono_create_INSERT_trigger(table, pk, current, history, fields, values)
+          func_name = quote_identifier_name(prefix: "chronomodel_", table: table, suffix: "_insert")
```

Comment:
Can we have this as a method?

Also with a couple of constants like `FUNCTION_PREFIX = 'chronomodel'` and `FUNCTION_SEPARATOR = '_'`

This method should work like `function_name(table, 'insert')` and return `chronomodel_table_insert`